### PR TITLE
Fix INVALID_OPERATION during initialization

### PR DESCRIPTION
**Pull request by at-wat**
`Enable/DisableVertexAttribArray()` is not automatically switched by `UseProgram` and must be manually switched before use.

### CODE REVIEW

**Comment by at-wat** [comment]
@seqsense/cloud_engineer please take a look

File: main_js.go [hunk 1]
```diff
@@ -688,13 +678,15 @@ func (pe *pcdeditor) runImpl(ctx context.Context) error {
...
 				gl.Disable(gl.BLEND)
+				clean()
 			}
 
 			if show2D && has2D {
 				// Render 2D map
 				gl.Enable(gl.BLEND)
 				gl.BlendFunc(gl.SRC_ALPHA, gl.ONE_MINUS_SRC_ALPHA)
 				gl.UseProgram(programMap)
+				enableVertexAttribs(gl, aVertexPosition, aTextureCoordMap)
```
Comment (f-fl0) [inline main_js.go:689]: Shouldn't clean be called as well here?

Comment (at-wat) [reply main_js.go]: good catch, it should

**Review by f-fl0 — APPROVED** [review]
LGTM

**Pull request merged**